tests/Controller/CRUDControllerTest.php

```diff
@@ -312,18 +306,6 @@ static function (string $name, object $object, array $parameters = []): string {
         }
     }
 
-    protected function tearDown(): void
-    {
-        parent::tearDown();
-
-        if (!$this->httpMethodParameterOverride && Request::getHttpMethodParameterOverride()) {
-            $disableHttpMethodParameterOverride = \Closure::bind(static function (): void {
```

Comment:
I don't know what does this code, but I have debug a little bit and I don't see any difference on the value of httpMethodParameterOverride before and after this code executed.

Reply:
It was introduce here: https://github.com/sonata-project/SonataAdminBundle/pull/6320.

It seems related to the config
```
framework:
    http_method_override: true
```

which is not necessary anymore since https://github.com/sonata-project/SonataAdminBundle/pull/7348 IMHO

Reply:
Nice then it is good to have it removed.